docs/UserGuide.md

```diff
@@ -239,20 +253,17 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd student in the address book.
 * `find n/Betsy` followed by `delete 1` deletes the 1st student in the results of the `find` command.
 
-### Receiving payment from a student : `pay`
+### Reminder : `remind`
```

Comment:
I thought this would be more suitable if it is also in the format of `xxxxing Reminder`. If you check through the UG, all the features are in this format.
Perhaps `Getting reminder for today` or anything better that you can think of?

Reply:
Thank you for pointing this out! Ive edited the UG to Getting a reminder for today